wolf-test-utils/src/Wolf/Data/Export/Types/Gen.hs

```diff
@@ -61,3 +61,20 @@ instance GenValid Repo where
             fmap M.fromList $ forM noteUuids $ \uuid -> (,) uuid <$> genValid
         repoSuggestions <- genValid
         pure Repo {..}
+
+instance (GenUnchecked a, GenUnchecked w, GenUnchecked e) =>
```

Comment:
This should really be in the library, but since you're pinning its version, that's fine.

Reply:
Do you want me to add a genvalidity dependency to a library?